web-client/src/presenter/presenter.ts

```diff
@@ -1291,7 +1291,10 @@ export const presenterSequences = {
     showThirtyDayNoticeModalSequence as unknown as Function,
   showViewPetitionerCounselModalSequence:
     showViewPetitionerCounselModalSequence as unknown as Function,
+  signOutIdleSequence: signOutIdleSequence as unknown as Function,
```

Comment:
Sequences should not be typed with "as unknown as Function" as that is only for legacy functions. 

You can look at web-client/src/presenter/sequences/sortTableSequence.ts for an example on how to type sequences.